site/content/en/docs/Installation/Creating Cluster/aks.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Azure Kubernetes Service"
+linkTitle: "Azure"
+weight: 30
+description: >
+  Follow these steps to create a cluster and install Agones directly on [Azure Kubernetes Service (AKS) ](https://docs.microsoft.com/azure/aks/).
```

Comment:
nit: Should it be "Kubernetes cluster" instead of "cluster"?

Reply:
Updated all the descriptions to be consistent, and more descriptive :+1: